### PR TITLE
supporters logos and copy for supporters page

### LINKED ISSUE #53: Add Supporters page
## Description
Add copy and markup to Supporters page

## Plan
- [ ] Add the copy & markup as per #13 

## Acceptance Criteria
- [x] Supporters info as per #13 can be reached
- [x] Copy signed off in situ by @katquatermass

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #53 

## Description

- Added supporters logos
- Added copy to page

Hopefully its all good 🤞 🐧

### CODE REVIEW

File: src/Views/Pages/Supporters.elm [hunk 1]
```diff
@@ -3,8 +3,9 @@
 
 module Views.Pages.Supporters exposing (supportersContent)
 
-import Html exposing (Html, div, h2, p, text)
-import Html.Attributes exposing (class)
+import Assets exposing (AssetPath(..), path)
+import Html exposing (Html, div, h2, p, text, img)
```
Comment (katjam) [inline src/Views/Pages/Supporters.elm]: There is a tool that does this for us (elm-analyse or I thought new version of format would also) - so it won't matter... but they should be in alphabetical order (with Caps ones first) which will make code review/ explicit changes easier to spot in future.

Please change for now manually

**Review by katjam — CHANGES_REQUESTED** [review]
Alphabetise import inclusions - otherwise beautiful!

File: src/Views/Pages/Supporters.elm (hunk 1 above)
Comment (HedgehogHolly) [reply src/Views/Pages/Supporters.elm]: I think I've understood what you meant and made the changes?

**Review by katjam — APPROVED** [review]
Prefect. I'm so excited about this page. :octocat:

**Pull request merged**